docs/DafnyRef/Attributes.md

```diff
@@ -619,17 +619,17 @@ until we succeed proving them, or there is only one
 single assertion that it timeouts (in which
 case an error is reported for that assertion).
 Defaults to 1.
-If [`{:vcs_split_on_every_assert}`](#sec-vcs_split_on_every_assert) is set, then this parameter is useless.
+If [`{:isolate_assertions}`](#sec-isolate_assertions) is set, then this parameter is useless.
 
 ### 11.2.24. `{:vcs_max_splits N}` {#sec-vcs_max_splits}
 
 Per-method version of the command-line option `/vcsMaxSplits`.
 Maximal number of [assertion batches](#sec-assertion-batches) generated for this method.
 In [keep-going mode](#sec-vcs_max_keep_going_splits), only applies to the first round.
 Defaults to 1.
-If [`{:vcs_split_on_every_assert}`](#sec-vcs_split_on_every_assert) is set, then this parameter is useless.
+If [`{:isolate_assertions}`](#sec-isolate_assertions) is set, then this parameter is useless.
 
-### 11.2.25. `{:vcs_split_on_every_assert}` {#sec-vcs_split_on_every_assert}
+### 11.2.25. `{:isolate_assertions}` {#sec-isolate_assertions}
 Per-method version of the command-line option `/vcsSplitOnEveryAssert`.
```

Comment:
```suggestion
Per-method version of the command-line option<span id="sec-vcs_split_on_every_assert"></span> `/vcsSplitOnEveryAssert`
```
You can keep the previous ID around just in case someone happened to put a hard link to the section from their blog post, their course or their PDF. That way the link is valid and relevant. I tested it and it works on the documentation page at least, when I do `location.hash = "#sec-vcs_split_on_every_assert"` it redirect to that paragraph now.

Reply:
Nice attention to detail